docs/en/operations/table_engines/index.md

```diff
@@ -2,15 +2,77 @@
 
 The table engine (type of table) determines:
 
-- How and where data is stored, where to write it to, and where to read it from.
+- How and where data is stored, where to write it to, and from where to read it.
 - Which queries are supported, and how.
 - Concurrent data access.
 - Use of indexes, if present.
 - Whether multithreaded request execution is possible.
 - Data replication parameters.
 
-When reading, the engine is only required to output the requested columns, but in some cases the engine can partially process data when responding to the request.
+## Engine families
 
-For most serious tasks, you should use engines from the `MergeTree` family.
+- `*MergeTree`
```

Comment:
I'd make those `###` subheaders instead of just list items (so they'll be part of ToC)